examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads
+-----
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a :class:`geopandas.GeoDataFrame` object.
+"""
+
+import geopandas as gpd
+import pygmt
+
+# Read shapefile data using geopandas
+gdf = gpd.read_file(
+    "http://www2.census.gov/geo/tiger/TIGER2015/PRISECROADS/tl_2015_15_prisecroads.zip"
+)
+# The dataset contains different road types listed in the RTTYP column,
+# here we select the following ones to plot:
+roads_common = gdf[gdf.RTTYP == "M"]  # Common name roads
+roads_state = gdf[gdf.RTTYP == "S"]  # State recognized roads
+roads_interstate = gdf[gdf.RTTYP == "I"]  # Interstate roads
+
+fig = pygmt.Figure()
+
+# Define target region around O'ahu (Hawai'i)
+region = [-158.3, -157.6, 21.2, 21.75]  # minx, maxx, miny, maxy
+
+fig.basemap(
+    region=region,
+    projection="M12c",
+    frame=["af", 'WSne+t"Main roads of Oahu (Hawaii)"'],
+)
```

Comment:
I've figured it out! We can use the octal code for apostrophe (') which is 047, see https://docs.generic-mapping-tools.org/6.2/cookbook/octal-codes.html.

```suggestion
title = r"Main roads of O\047ahu (Hawai\047i)"  # \047 is octal code for '
fig.basemap(region=region, projection="M12c", frame=["af", f'WSne+t"{title}"'])
```

Reply:
Great @weiji14!

Reply:
Realizing two years later that I used the wrong symbol :sweat_smile: It should be an [Okina](https://en.wikipedia.org/wiki/%CA%BBOkina) `ʻ` (U+02BB) instead of an apostrophe. Looks more like a 6 than a 9.